addon/components/infinity-loader.js

```diff
@@ -69,5 +73,14 @@ export default Ember.Component.extend({
     if (this.get('infinityModel.reachedInfinity') && this.get('destroyOnInfinity')) {
       this.destroy();
     }
+  }),
+
+  infinityModelPushed: Ember.observer('infinityModel.[]', function() {
```

Comment:
From my understanding, .[] is the same as .length, but I can't find confirmation. Any insight into this?